GSoC/index.html

```diff
@@ -0,0 +1,51 @@
+---
+layout: default
+title: "Information for Google Summer of Code"
+download: true
+---
+<h2>Google Summer of Code Information</h2>
+
+<h3>How to become a Open Live Writer Contributor</h3>
+<p>
+    <ul>
+        <li>
+            Set up a blog, several are suitable, see our documentation.
+        </li>
+        <li>
+            Download and install Open Live Writer
+        </li>
+        <li>
+            Use it to create a few blog posts
+        </li>
+        <li>
+            Join us in gitter.im  - <a href="https://gitter.im/OpenLiveWriter/OpenLiveWriter"> https://gitter.im/OpenLiveWriter/OpenLiveWriter</a>
+        <li>
+            Go through the issues, both open and closed.  That will give you an idea about 
+            what our user base cares about, and what features we have implemented so far
+        </li>
+        <li>
+            Now that you are familar with the issues, start answering new issues, and requesting information needed to solve the problem.  Duplicates are easy to resolve
+        </li>
+        <li>
+            Start reading the code, there are directions in the README.md
+        </li>
+        <li>
+            Fill out an application and indicate what project you would like to contribute to.  There are suggestions below, but we are open to other suggestions.
+        </li>
+</ul>
+</br>
+<h3>Project Ideas</h3>
+<p>These are all popularly requested features.</p>
+<ul>
+    <li>Portable version
+        <p>This would involve a fork of OpenLiveWriter so that it could run on 
+            a thumbdrive or similar device, allowing the user to use someone else's 
+            computer without modifying it.  See <a href=http://portableapps.com> http://portableapps.com/</a></p> 
+    </li>
+<li>YouTube Video (and other providers) includes
+    <p>A former feature of Live Writer, we need to add support of videos.  There are several YouTube videos showing this support</p>
```

Comment:
I would clarify this a little. It looks like there are two separate pieces of functionality that involve YouTube that seem to be broken right now. The first is 'Insert > Video > From website' - you used to be able to just paste a YouTube URL in there, but that didn't work for me when I tried it a minute ago. There is also 'Insert > Video > From video service...' - this requires you to login so that you can see your own account's videos. Which one were you thinking?